src/systems/animation.js

```diff
@@ -0,0 +1,31 @@
+var registerSystem = require('../core/system').registerSystem;
+var THREE = require('../lib/three');
+
+var camera = null;
+var scene = null;
+
+/**
+ * Animation system.
+ *
+ * Controls library-level animation events agnostic to implementations.
+ *
+ */
+module.exports.System = registerSystem('animation', {
+  init: function () {
+    this.sceneEl.addEventListener('camera-ready', function () {
+      scene = this.sceneEl.object3D;
+      camera = this.sceneEl.camera;
+    }, true);
+  },
+
+  /**
+   * Update animation mixers and shaders.
+   *
+   */
+  tick: function () {
+    if (scene && camera) {
+      THREE.glTFAnimator.update();

```

Comment:
There's also a global animation update in [a-scene.js::tick()](https://github.com/aframevr/aframe/blob/master/src/core/scene/a-scene.js#L286-L287) ... would it make sense to move that here, or vice versa?